vendor/vendor.json

```diff
@@ -2,94 +2,550 @@
 	"comment": "",
```

Comment:
You'll need to run make vendorfmt to fix formatting (our make dev also runs this for you(

Reply:
Made the change. Curious why you all do this?

Reply:
no more merge conflict hell, also its much easier to see and understand a diff at a glance, and it is sorted by path which is nice.

Reply:
good catch! yeah it makes merging the vendor file...possible - we have some extra stuff in the enterprise fork and it was a nightmare before `vendorfmt`

Reply:
Nice! Thanks for explaining 👍